Files/ViewModels/ItemViewModel.cs

```diff
@@ -819,9 +819,7 @@ private async Task LoadItemThumbnail(ListedItem item, uint thumbnailSize = 20, I
                         var matchingStorageFolder = (StorageFolder)matchingStorageItem ?? await GetFolderFromPathAsync(item.ItemPath);
                         if (matchingStorageFolder != null)
                         {
-                            var mode = thumbnailSize < 80 ? ThumbnailMode.ListView : ThumbnailMode.SingleItem;
```

Comment:
I noticed this change will make it try and show previews, which isn't useful since it's so small. Should this e reverted?
![image](https://user-images.githubusercontent.com/59544401/129666970-59a876e6-b0ad-49be-b9a0-4ba36a9db975.png)

Reply:
also, @gave92 you were right xD. It looks like ThumbnailOptions.ResizeThumbnail is the better option. `UseCurrentScale` won't use smaller icons like this one
![image](https://user-images.githubusercontent.com/59544401/129671046-2ae86279-b422-4825-a0e0-eeb61480cc2c.png)

Reply:
Does Resize cut off the thumbnail?